src/lib/integration_test_lib/graphql_requests.ml

```diff
@@ -0,0 +1,1013 @@
+open Core_kernel
+open Async
+open Mina_base
+open Mina_transaction
+
+(* exclude from bisect_ppx to avoid type error on GraphQL modules *)
+[@@@coverage exclude_file]
+
+let node_password = "naughty blue worm"
```

Comment:
This should probably be an argument rather than a hard-coded constant in this module

Reply:
i fixed your other nits, i think this particular thing i'll leave for a different PR, as it would take a little bit of design to figure out where this info should live and how it should flow.